### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by krakozavr** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#90] [General]
I just got a new computer with the fresh, brand-new, factory-installed Win11 22H2 build 22621.963
According to Microsoft, D'N'D should work with this build. But it doesn't :)

I came to Google first, and I found this app, but I decided to ask MS support first. 
It took about 4 hours when the support first confirmed that it should work; then applied a registry hack that killed the taskbar almost entirely; then they performed the repair upgrade installation; then - surprise!

> _"Since we are down to our last option, although we don't recommend it, since you badly need the feature for work, but please use third-party apps on your own risk as we cannot recommend any other non-Microsoft verified applications and programs. "_

So, I returned to my bookmarks and installed this little beast which solved all D'N'D problems - even the long-lived troubles with D'N'D between WIndowsINK enabled-and-disabled apps, which I had even in Winn10.

God bless Indiw Dewelopers :)

**Comment by HerMajestyDrMona** [comment]
Thank you for sharing your feedback!

I'm still surprised to read that certain users can't use drag and drop in 22H2. For me it sounds like they made a mess with enabled/disabled features. There is a program called "ViVeTool" that allows to enable/disable windows "features", so maybe it's worth to search for the command that enables drag and drop using it?

Either way, I'm extremely happy that this program still works fine after all these months. I think making it system independent was the right decision. It still won't work as perfectly as the native support (for example it can't determine mouse clicks from windows/programs that are running as administrator), however in most of daily-use cases if should be fine :)

**Reply by krakozavr** [reply]
And thank you again! It's still the only way to make things work on both my computers :)
Recently it stopped working again, after the Win 11 update, but with the DetectKnownPixelColorsToPreventAccidentalEvents=0 hack (obviously intended for some non-standard situations) - and it is back in business :)
